3rdParty/fuerte/src/GeneralConnection.h

```diff
@@ -103,6 +103,7 @@ class GeneralConnection : public fuerte::Connection {
   // Start an asynchronous request.
   void sendRequest(std::unique_ptr<Request> req, RequestCallback cb) override {
     // construct RequestItem
+    req->setTimeReceived();
```

Comment:
Nit: although this is correct, the wording really confused me.
Because the request is not received here in the sense of Request -> Response (which could be received), but rather the request is received by the fuerte subsystem for processing.
I suggest to rename this to `setTimeProcessingStarted()` or `setTimeStarted()` or something.